keel-core/src/main/kotlin/com/netflix/spinnaker/keel/diff/DefaultResourceDiff.kt

```diff
@@ -71,13 +77,23 @@ data class DefaultResourceDiff<T : Any>(
     val differ: ObjectDiffer = ObjectDifferBuilder
       .startBuilding()
       .apply {
+        comparison().ofType(Instant::class.java).toUseEqualsMethod()
```

Comment:
I noticed the differ is doing a field-by-field diff of `java.time.Instant` which doesn't make sense and will obfuscate the diff output